### PR TITLE
remove TODO + change for Rom: placeholder until Sunday.

### PR DESCRIPTION
**Pull request by yourgirlcodes**

### CODE REVIEW

File: CDP.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export type DataCenter = 'eu5' | `us5` | `il1`;
 type StagingEnvs = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8;
 export type Env<n extends StagingEnvs = StagingEnvs> = 'prod' | `st${n}`;
 export const availableEnvs: Record<DataCenter, Env[]> = {
+    us5: [], // WIP: Rom B.
```
Comment (iBaryo) [inline CDP.ts]: * change to `us5: ['prod', ...createArray(1, n => `st${n + 1}` as Env)]`
* remove comment

File: CDP.ts [hunk 2]
```diff
@@ -158,9 +159,6 @@ export class CDP {
     }
 
     private getDomainDc({dataCenter, env} = this.options) {
-        if (dataCenter != 'il1' && env == 'prod')
-            return dataCenter;
-
```
Comment (iBaryo) [inline CDP.ts]: do not remove this - this is to make sure that when requesting a `prod` env, then it's being dropped from the domain. it's relevant for eu5 and us5

File: entities/Connector/Auth.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import {JSONSchema7} from "json-schema";
 import {WithType} from "../common";
 
-// @ts-ignore //TODO: ask Baryo + remove
+// @ts-ignore
```
Comment (iBaryo) [inline entities/Connector/Auth.ts]: remove this and solve the issue

File: CDP.ts (hunk 2 above)
Comment (kobikas) [reply CDP.ts]: you right , anyway i checked and the api's works on both domains ( cdp.eu5.gigya.com and  cdp.eu5-prod.gigya.com- is it ok? )
so @zoe , pls keep it as it was.

**Pull request merged**